### DISCUSSION TITLE
Welcome to Fast-DDS Discussions!

### DISCUSSION THREAD

**Discussion by EduPonz** [eProsima/Fast-DDS#2390] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JaganKumar-TRM** [comment]
Without using the governance file, How to programmatically set the properties "liveliness_protection_kind" & "discovery_protection_kind" for DDS participants?.
Because there is no documentation for that on the “https://fast-dds.docs.eprosima.com” site.

**Reply by EduPonz** [reply]
Hi @JaganKumar-TRM ,

It'd be great if you could open a question in [Q&A](https://github.com/eProsima/Fast-DDS/discussions/categories/q-a) so that others can find it and refer to it.

**Reply by JaganKumar-TRM** [reply]
Thanks for the response @EduPonz .
I will raise this in the Q&A section.

**Comment by swarm59** [comment]
Hi, my name is Francesco Fedi and I am currently evaluating the python version of fastDDS to control a Swarm of unmanned vehicles based on ROS2.

**Reply by MiguelCompany** [reply]
Hi @swarm59!

We hope you have a nice experience with FastDDS!

**Comment by GeziP** [comment]


**Reply by JLBuenoLopez** [reply]
Hi @GeziP 

It would be nice if you could open a ticket in the [Support](https://github.com/eProsima/Fast-DDS/discussions/categories/support) discussion forum so each use case has its own ticket and there is no mixed information in the same thread. Thanks for your understanding!
